### PR TITLE
fix(react-native-adapter): Bare React Native app _reactNative.UIManager.getViewManagerConfig is not a function close #4737

### LINKED ISSUE #4737: [expo-av] Bare React Native app _reactNative.UIManager.getViewManagerConfig is not a function
### Environment

```
  Expo CLI 2.20.10 environment info:
    System:
      OS: macOS High Sierra 10.13.6
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.16.0 - /usr/local/bin/node
      Yarn: 1.8.0 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.3 AI-182.5107.16.33.5264788
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.3 => 16.8.3
      react-native: 0.55.4 => 0.59.8
    npmGlobalPackages:
      expo-cli: 2.20.10
```

```
"react-native-unimodules": "0.4.1"
"expo-av": "^5.0.2",
```

### Steps to Reproduce

add code in js under :

```
import { Audio } from 'expo-av'
console.log(Audio)
```

### Expected Behavior

run normal

### Actual Behavior

image

### PR DESCRIPTION
**Pull request by Pines-Cheng**
# Why

Accessing view manager configs directly off UIManager via UIManager[ ] is no longer supported from React Native 0.58. Use UIManager.getViewManagerConfig() instead.

Which cause Bare React Native app _reactNative.UIManager.getViewManagerConfig is not a function  #4737 

# How

add a line of compatible code

# Test Plan

use expo-av under React Native@0.55.4 , It's fine.

### CODE REVIEW

File: packages/@unimodules/react-native-adapter/src/NativeViewManagerAdapter.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function requireNativeViewManager<P = any>(viewName: string): React.Compo
   const reactNativeViewName = `ViewManagerAdapter_${viewName}`;
   const ReactNativeComponent = requireNativeComponent(reactNativeViewName);
   // @ts-ignore: UIManager.getViewManagerConfig is not declared
-  const reactNativeUIConfiguration = UIManager.getViewManagerConfig(reactNativeViewName) || {
+  const reactNativeUIConfiguration = (UIManager.getViewManagerConfig ? UIManager.getViewManagerConfig(reactNativeViewName):UIManager[reactNativeViewName]) || {
```
Comment (tsapeta) [inline packages/@unimodules/react-native-adapter/src/NativeViewManagerAdapter.tsx]:
```suggestion
  const reactNativeUIConfiguration = (UIManager.getViewManagerConfig ? UIManager.getViewManagerConfig(reactNativeViewName) : UIManager[reactNativeViewName]) || {
```

**Review by tsapeta — CHANGES_REQUESTED** [review]
Hey @Pines-Cheng, thanks for contributing! 💪 
This looks good to me, but could you please apply this one formatting change, run `yarn build` in `react-native-adapter` package and commit changes it made? JavaScript files in `build` folder will be updated as well after that 🙂

**Comment by Pines-Cheng** [comment]
have formatting code and run yarn build in react-native-adapter~

File: packages/@unimodules/react-native-adapter/build/index.d.ts [hunk 2]
```diff
@@ -5,6 +5,6 @@ import SyntheticPlatformEmitter from './SyntheticPlatformEmitter';
 import { CodedError } from './errors/CodedError';
 import { UnavailabilityError } from './errors/UnavailabilityError';
 export declare const Platform: {
-    OS: "ios" | "android" | "windows" | "macos" | "web";
+    OS: import("react-native").PlatformOSType;
```
Comment (tsapeta) [inline packages/@unimodules/react-native-adapter/build/index.d.ts]: I don't think we want this change 🤔

```suggestion
    OS: "ios" | "android" | "windows" | "macos" | "web";
```

**Review by tsapeta — CHANGES_REQUESTED** [review]

**Comment by Pines-Cheng** [comment]
thank you for review~

**Review by tsapeta — APPROVED** [review]
Thank you so much for your work 🙂

**Pull request merged**